src/test/java/es/princip/getp/domain/people/controller/ProjectLikeControllerTest.java

```diff
@@ -90,5 +91,37 @@ void like_WhenPeopleIsNotFound_ShouldBeFailed() throws Exception {
                 .contentType(MediaType.APPLICATION_JSON))
                 .andExpect(status().isNotFound());
         }
+
+        @DisplayName("의뢰자는 피플에게 좋아요 취소를 할 수 있다.")
```

Comment:
```suggestion
        @DisplayName("의뢰자는 피플에게 좋아요를 취소할 수 있다.")
```
거기서 거기인 것 같지만, 주석 어투를 통일하면 좋겠다는 생각에..!